### ISSUE TITLE
Nodes out of sync?

### ISSUE THREAD

**Issue by Mandelhoff** [zack-bitcoin/amoveo#71]
The 159.89.106.253 node was down for a few hours. During that time, I mined a few blocks on my node. My node is at height 105.

The 159.89.106.253 node is up now, and has block height 99. I tried a sync:start(). on my node, and got the "weird response" messages shown in this screenshot.

![image](https://user-images.githubusercontent.com/35471470/35346304-8070ff6e-00f7-11e8-9f93-4297b6862ada.png)

Will 159.89.106.253 sync up with my height at 105, or do I have to do something to sync back to 159.89.106.253 at height 99?

**Comment by zack-bitcoin** [comment]
Thank you for sharing this.
There was a bug in the recent_blocks.erl code. 
This code is used to know what should be pruned and when.
The node was unable to sync your blocks because it would crash when it tried pruning the old info.

I will fix this soon and report back here when I do.

**Comment by zack-bitcoin** [comment]
I fixed a bunch of stuff today. Maybe it will work now?
If not, I recommend turning off the node. doing 'make prod-clean' to delete the blocks. then turn it back on and re-sync.

**Comment by Mandelhoff** [comment]
My local node is pretty messed up right now. I tried a code update (git pull) and then rebuilt my node. It still showed my height, but it wouldn't sync.

Next, I tried the 'make prod-clean'. Same - no sync.

Next, I tried "rm -rf amoveo" and a clean git clone, build, etc... - but, the fresh clone couldn't attach due to "Node is not running!" like this:

![image](https://user-images.githubusercontent.com/35471470/35366545-a5d8db4e-013f-11e8-86c7-3b5d4666a3df.png)

I'll keep trying tomorrow.
